extra-dev/packages/coq-vst/coq-vst.dev/opam

```diff
@@ -1,22 +1,36 @@
 opam-version: "1.2"
 authors: ["Andrew W. Appel"
-          "Lennart Beringer"
-          "Jean-Marie Madiot"
-          "Qinxiang Cao"
-          "Santiago Cuellar"
-          "Nick Giannarakis"
-         ]
-maintainer: "matej.kosik@inria.fr"
-homepage: "http://vst.cs.princeton.edu/download/"
+  "Lennart Beringer"
+  "Sandrine Blazy"
+  "Qinxiang Cao"
+  "Santiago Cuellar"
+  "Robert Dockins"
+  "Josiah Dodds"
+  "Nick Giannarakis"
+  "Samuel Gruetter"
+  "Aquinas Hobor"
+  "Jean-Marie Madiot"
+  ]
+maintainer: "VST team"
+homepage: "http://vst.cs.princeton.edu/"
 dev-repo: "https://github.com/PrincetonUniversity/VST.git"
 bug-reports: "https://github.com/PrincetonUniversity/VST/issues"
 license: "https://raw.githubusercontent.com/PrincetonUniversity/VST/master/LICENSE"
 build: [
-  [make "IGNORECOQVERSION=true" "-j%{jobs}%"]
-]
-install: []
-remove: []
+	[make "-j%{jobs}%" "COMPCERT=%{lib}%/coq/user-contrib/compcert" ".loadpath" "version.vo" "msl" "veric" "floyd"]
+	]
+install: [
+	["mkdir" "-p" "%{lib}%/coq/user-contrib/VST"]
+	["cp" "-r" "msl" "%{lib}%/coq/user-contrib/VST/"]
+	["cp" "-r" "veric" "%{lib}%/coq/user-contrib/VST/"]
+	["cp" "-r" "floyd" "%{lib}%/coq/user-contrib/VST/"]
+	["cp" "-r" "sepcomp" "%{lib}%/coq/user-contrib/VST/"]
+	]
+remove: [
+	["rm" "-fr" "%{lib}%/coq/user-contrib/VST"]
+	]
 depends: [
-  "coq" {= "dev"}
+  "coq" {>= "8.8.0"}
```

Comment:
Do we need such late versions of Coq and CompCert?

Reply:
@liyishuai Indeed, we should allow  `>= 8.7.0`.
Will update that.

@lastland At the moment, this PR is useless, this pull request will be usefull as soon as this other one  ( https://github.com/AbsInt/CompCert/pull/228 ) is accepted.
**CompCert does not install any `.vo` files at the moment.** As such the opam package of CompCert is installing the compiler binaries but is completely useless for any coq developer as the `.vo` files are not accessible. :disappointed: